lib/exrabbit/util.ex

```diff
@@ -87,6 +87,9 @@ defmodule Exrabbit.Util do
   end
 
   defp get_formatter_module(atom) do
-    Module.concat([Exrabbit.Formatter, Atom.to_string(atom) |> String.upcase()])
```

Comment:
Protocols do all that magic internally and hide that from us, which is not true in this case - we manually create new modules with a prefix that related to the 3rd library (exrabbit).
Also, I wonder how it would behave in case when two applications introduces two formatters for the same type. I guess it would not compile.
Thought I highly doubt that any `Exrabbit.Formatter` will be published (except the most popular and obvious one), this change doesn't break this capability, but only extend it. It just allows you to write your own custom formatters without obligating users to write wrappers.